package/cpp/api/JsiSkTypeFaceFontProviderFactory.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <memory>
+#include <utility>
+
+#include <jsi/jsi.h>
+
+#include "JsiSkData.h"
+#include "JsiSkHostObjects.h"
+#include "JsiSkTypefaceFontProvider.h"
```

Comment:
@wcandillon I not sure how much is C++ case sensitive but there seems to be difference in casing: JsiSk**Typeface**FontProvider.h vs JsiSk**TypeFace**FontProvider.h